repositories/SectionRepository.js

```diff
@@ -99,3 +120,52 @@ module.exports.duplicateSection = function duplicateSection(id, position) {
     });
   });
 };
+
+module.exports.createIntroduction = function createIntroduction(
+  introductionInput
+) {
+  return db.transaction(async trx => {
+    await trx("Sections")
+      .update({ introductionTitle: null, introductionContent: null })
+      .where({ id: introductionInput.sectionId });
+
+    return trx("Sections")
+      .update({
+        introductionTitle: introductionInput.title,
+        introductionContent: introductionInput.content,
+        introductionEnabled: true
+      })
+      .where({ id: introductionInput.sectionId })
+      .returning("*");
+  });
+};
+
+module.exports.updateIntroduction = async function updateIntroduction({
+  sectionId,
+  title,
+  content
+}) {
+  await Section.update(sectionId, {
+    introductionTitle: title,
+    introductionContent: content
+  });
+
+  return {
+    title,
+    content
```

Comment:
Could we return the result from the database here rather than the input args, would add bit more assurance the data has entered the db correctly?